samples/packages.go

```diff
@@ -0,0 +1,8 @@
+// Packages
+package main
+
+import "fmt"
```

Comment:
Yeah, I was contemplating prefixing them with `01_` etc. but then I thought: too much effort if I decide later I need to add another one in between, or should I update?

Reply:
as per previous comment, i had not found the association between the slides and these files. don't number the files for the reason you gave. but i still think this sample should import multiple packages since packages is the topic (not package :) )